sdk/nodejs/provider/provider.ts

```diff
@@ -158,7 +158,7 @@ export interface Provider {
      * @param olds The old input properties to use for validation.
      * @param news The new input properties to use for validation.
      */
-    check?: (urn: resource.URN, olds: any, news: any, sequenceNumber: number) => Promise<CheckResult>;
+    check?: (urn: resource.URN, olds: any, news: any) => Promise<CheckResult>;
```

Comment:
Are `sequenceNumbers` strictly not needed here? Or should we keep the parameter but make it optional? (which also may avoid the breaking change vs. current master)